src/inspector_agent.cc

```diff
@@ -243,6 +243,10 @@ class NodeInspectorClient : public v8_inspector::V8InspectorClient {
     return uv_hrtime() * 1.0 / NANOS_PER_MSEC;
   }
 
+  void contextDestroyed(Local<Context> context) {
+    inspector_->contextDestroyed(context);
```

Comment:
Inspector is supposed to live past the context lifetime (e.g. some tools like profiler can still work with the data it gathered).

I removed contextCreated notification from the constructor. I am trying to implement multicontext and it seems more logical if contextCreated and contextDestroyed are called from outside.